amandas-project/src/containers/Game.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import Cell from '../components/Cell'
+
+const CELL_SIZE = 20
+const WIDTH = 400
+const HEIGHT = 400
+
+class Game extends React.Component {
```

Comment:
I would recommend using functional components with hooks, rather than class components. They are more modern, cleaner and are currently more widely used.